projects/plugins/jetpack/modules/subscriptions.php

```diff
@@ -960,7 +960,9 @@ public function maybe_set_first_published_status( $new_status, $old_status, $pos
 	 * @return bool
 	 */
 	public function first_published_status_meta_auth_callback() {
-		if ( current_user_can( 'publish_posts' ) ) {
+		$capability = apply_filters( 'jetpack_subscriber_was_published_capability', 'publish_posts' );
```

Comment:
What do you think about making the filter a bit more specific, since it will only apply to that specific use check? Maybe something like `jetpack_subscriptions_post_was_ever_published_capability`? It's a mouthful, but it will avoid confusion for folks seeing that filter in the wild in the future.

Reply:
Agree, I was trying to figure out how to name a filter in a manner that was descriptive but also balanced the length. I'm fine with your suggestion.